docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Reviews Rating
+---
+
+The following documentation goes through an example implementation of a review rating platform for the food service industry on Sui. By leveraging blockchain to allow reviewers to own and monetize their feedback, the on-chain platform ensures transparency and temper-proof ratings.
```

Comment:
```suggestion
The following documentation goes through an example implementation of a review rating platform for the food service industry on Sui. By leveraging blockchain to allow reviewers to own and monetize their feedback, the on-chain platform ensures transparency and tamper-proof ratings.
```

Small typo fix in suggestion, but adding to the comment about detail here, I'm not sure I follow this last sentence: Why does allowing reviewers to own and monetize their feedback ensure transparency and tamper-proof ratings? To me these read like unrelated benefits (using a blockchain for storing reviews and ratings allows reviewers to own and monetize their feedback *and* ensures transparency and tamper-proof ratings.

Reply:
Also, can you add some details here on what concepts from Sui someone will encounter or learn about over the course of this guide?